pkg/kubelet/kubelet_pods.go

```diff
@@ -1625,17 +1625,17 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 			ImageID:      cs.ImageID,
 			ContainerID:  cid,
 		}
-		switch cs.State {
```

Comment:
i'd prefer to keep the `switch cs.State` and just add an `if` to the `ContainerStateExited` case

Reply:
> i'd prefer to keep the `switch cs.State` and just add an `if` to the `ContainerStateExited` case

tried the other factorization and we agreed that it was slightly more awkward